docker/DockerSetup.md

```diff
@@ -33,12 +33,11 @@ After the above, you can open a command line, navigate to the `docker` directory
 ### Running the Docker Container
 
 Open a command line (bash (Ubuntu) or Powershell (Windows)) and navigate to the repo directory. `Run the start_docker_[OS].x` script. This should start a container. You can do this in additional terminals to connect to the same container as well.
-
-
+Incase you encounter 'running scripts is disbaled on this system' error, for windows follow the top answer here: https://stackoverflow.com/questions/4037939/powershell-says-execution-of-scripts-is-disabled-on-this-system 
```

Comment:
While you're at it, there are a couple of typos here. Optionally, I think adding this as a new paragraph makes it a bit more readable (extra newline).
```suggestion

In case you encounter 'running scripts is disabled on this system' error, for windows follow the top answer here: https://stackoverflow.com/questions/4037939/powershell-says-execution-of-scripts-is-disabled-on-this-system 
```